packages/office-ui-fabric-react/src/components/FloatingPicker/BaseFloatingPicker.tsx

```diff
@@ -157,6 +151,9 @@ export class BaseFloatingPicker<T, P extends IBaseFloatingPickerProps<T>>
 
   protected renderSuggestions(): JSX.Element | null {
     const TypedSuggestionsControl = this.SuggestionsControlOfProperType;
+    if (this.props.suggestionItems) {
+      this.suggestionStore.updateSuggestions(this.props.suggestionItems!);
```

Comment:
Depending on how expensive this is, you might not want to do it every render. A potential alternative would be either memoizing the `updateSuggestions` call, or adding something like this to `componentDidUpdate`. (Any re-renders done here will be flushed before the displayed DOM is updated.)
```ts
if (this.props.suggestionItems && prevProps.suggestionItems !== this.props.suggestionItems) {
  this.suggestionStore.updateSuggestions(this.props.suggestionItems);
  this.forceUpdate();
}
```

Reply:
This really isn't too expensive and just makes sure everything is in sync. I think that this should stay here.